feat(cli): add docs (#4837)
microsoft/playwright: docs/cli.md

# Playwright CLI

> Playwright comes with a few command line tools that can be run with with `npx` or in `npm` scripts.

Reviewer: ```suggestion
Playwright comes with the command line tools that run via `npx` or inside `npm` scripts.
```
